VideoCallKitQuickStart/ViewController.swift

```diff
@@ -80,6 +77,13 @@ class ViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         self.title = "QuickStart"
+
+        /*
+         * The important thing to remember when providing a TVIAudioDevice is that the device must be set
+         * before performing any other actions with the SDK (such as creating Tracks, or connecting to a Room).
+         * In this case we've already initialized our own `TVIDefaultAudioDevice` instance which we will now set.
+         */
+        TwilioVideo.audioDevice = self.audioDevice;
```

Comment:
Personally I find it confusing that we call `prepareLocalMedia()` before joining a Room, but not in `viewDidLoad`. I kept the same pattern but it means that I can't set the audio device in the method or an exception will be thrown on a real device since the TVILocalVideoTrack is already created by that time.